Marlin/src/module/planner.cpp

```diff
@@ -1650,6 +1650,24 @@ void Planner::quick_stop() {
   stepper.quick_stop();
 }
 
+#if ENABLED(REALTIME_REPORTING_COMMANDS)
+
+  void Planner::quick_pause() {
+    // Suspend until quick_resume is called
+    // Don't empty buffers or queues
+    const bool did_suspend = stepper.suspend();
+    if (did_suspend)
+      TERN_(FULL_REPORT_TO_HOST_FEATURE, set_and_report_grblstate(M_HOLD));
```

Comment:
I would enjoy hearing some audio samples of the horrific sounds the printer makes when you stop it dead in the middle of a fast move. These audio samples can be included as part of the documentation.

Reply:
lol...
They would be exactly the same as when you press an end stop switch while moving fast, if the feature is active in your config. Uses the same stop mechanism.